Sequencer.cs

```diff
@@ -123,32 +123,35 @@ public void Save(PasswordSequenceConfiguration configuration)
              */
         }
 
-        public string GenerateSequence(PasswordSequenceConfiguration globalConfiguration, Random randomSeed)
+        public string GenerateSequence(PasswordSequenceConfiguration globalConfiguration, CryptoRandomRange cryptoRandom)
         {
             string targetSequence = string.Empty;
             foreach (SequenceItem sequenceItem in globalConfiguration.Sequence)
-                if (sequenceItem.Probability != PercentEnum.Never && randomSeed.Next(101) <= (int)sequenceItem.Probability)
-                    targetSequence += GenerateSequenceItem(sequenceItem, globalConfiguration, randomSeed);
+                if (sequenceItem.Probability != PercentEnum.Never && (int)cryptoRandom.GetRandomInRange(0, 100) <= (int)sequenceItem.Probability)

```

Comment:
Should probably split the line in two (I know it wasn't done before :/ ) as you did mostly everywhere else; ie: better readability